docs/architecture/application/tasks.md

```diff
@@ -68,26 +91,38 @@ def upgrade() -> None:
     pass
 ```
 
-This just needs to add an entry in the workflows table. No relations with other tables like how the workflow id gets a relation in the products table and etc.
+Note that, unlike a normal workflow:
 
-### Running the task in the UI
+* `is_task` is set to `True`.
+* No relations with other tables are created. (For example, a normal workflow id gets a relation in the products table, etc.)
```

Comment:
Since you modified this :) I suggest we change this section to redirect users to "Registering Workflow"  https://workfloworchestrator.org/orchestrator-core/getting-started/workflows/#register-workflows which explains to either use the generator or the migration helper function. That page would need to be modified slightly to add that there's also a `create_task` helper in `orchestrator.migrations.helpers`

Reply:
This rearrangement seems like a larger conversation, so marking this resolved with respect to the current MR.